ucFirst/ucFirst.js

```diff
@@ -0,0 +1,5 @@
+String.prototype.ucFirst = function() {
+  return String.fromCharCode(this.charCodeAt(0) & 223) + this.substr(1);
```

Comment:
This line of code is not really readable as the reader would have to evaluate it in his brain before arriving at what the code base does. Split it to several lines and add descriptive comments in the function.

Reply:
But you said to achieve it with a one-liner